README.md

```diff
@@ -6,6 +6,7 @@ and Linux. Please visit [xqemu.com](http://xqemu.com) to learn more.
 
 Build Status
 ------------
-| Windows | macOS | Linux |
-| ------- | ----- | ----- |
-| [![Build status](https://ci.appveyor.com/api/projects/status/i1m0keigjabxg170/branch/master?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu) | [![Travis-CI Status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/2)](https://travis-ci.org/xqemu/xqemu) | [![Travis-CI Status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/1)](https://travis-ci.org/xqemu/xqemu) |
+| Build | Windows | macOS | Linux |
+| ----- | ------- | ----- | ----- |
+| Debug | [![Build status](https://ci.appveyor.com/api/projects/status/i1m0keigjabxg170/branch/master?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu/branch/master) | [![Travis-CI Status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/3)](https://travis-ci.org/xqemu/xqemu) | [![Travis-CI Status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/1)](https://travis-ci.org/xqemu/xqemu) |
```

Comment:
nit: Somehow labels for the Travis and AppVeyor images are different:

`![Build status]` vs. `![Travis-CI Status]`

We should make this more consistent while you are at it.

Reply:
I force pushed it

Reply:
Approved. Go for merge.